service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -149,9 +149,9 @@ void shouldMakeCorrectCallForAllocatedJudgeNotificationC2Application(Boolean tog
     void shouldMakeCorrectCallForNewCaseStateModel(Boolean toggleState) {
         givenToggle(toggleState);
 
-        assertThat(service.isNewCaseStateModelEnabled()).isEqualTo(toggleState);
+        assertThat(service.isCaseUserBulkAssignmentEnabled()).isEqualTo(toggleState);
```

Comment:
does not looks right

Reply:
thanks, duplicate while refactoring. Removed